ui/client/components/graph/node-modal/editors/Validators.ts

```diff
@@ -59,7 +67,9 @@ export function allValid(validators: Array<Validator>, values: Array<string>): b
   return withoutDuplications(validators).every(validator => validator.isValid(...values))
 }
 
-export const validators: Record<ValidatorName, (errors?: Array<Error>, fieldName?: string) => Validator> = {
-  [ValidatorName.MandatoryParameterValidator]: () => mandatoryValueValidator,
-  [ValidatorName.ErrorValidator]: (errors, fieldName) => errorValidator(errors, fieldName),
+export const validators: Record<HandledErrorType, (errors?: Array<Error>, fieldName?: string) => Validator> = {
+  [HandledErrorType.MandatoryParameterValidator]: () => mandatoryValueValidator,
+  [HandledErrorType.NotBlankParameterValidator]: () => notBlankValueValidator,
+  [HandledErrorType.ErrorValidator]: (errors, fieldName) => errorValidator(errors, fieldName),
+  [HandledErrorType.WrongDateFormat]: (errors, fieldName) => errorValidator(errors, fieldName),
```

Comment:
why WrongDateFormat is mapped to errorValidator?

Reply:
removed